backend/src/main/java/org/sefglobal/core/fellowship/controller/admin/CertificateAdminController.java

```diff
@@ -34,4 +34,10 @@ public Certificate addCertificate(@RequestBody Certificate certificate){
     public String deleteCertificateById(@PathVariable Integer id) throws ResourceNotFoundException {
         return certificateService.deleteCertificate(id);
     }
+
+    @PutMapping("/{id}")
+    @ResponseStatus(HttpStatus.OK)
+    public Certificate updateCertificateById(@PathVariable Integer id, @RequestBody Certificate certificate) throws ResourceNotFoundException {
```

Comment:
Hi let's format this sentence it's too long.
```suggestion
    public Certificate updateCertificateById(@PathVariable Integer id, 
                                             @RequestBody Certificate certificate) 
            throws ResourceNotFoundException {
```

Reply:
Sure!

Reply:
Resolved with commit [24d10be](https://github.com/sef-global/sef-core/pull/99/commits/24d10be28040012d2f837a232501ddd5d7424d52)